src/instructlab/model/chat.py

```diff
@@ -61,6 +62,27 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = cfg.get_api_base(cfg._serve().host_port)
```

Comment:
(No action required) You could also move the default value for `host_port` into a variable, then you wouldn't need to call `_serve()` here.

Reply:
We can probably put it in the `DEFAULTS` object. But let's not do that in this PR, we should really just get this merged so we can resolve the original issue we wanted to fix - which is that `ilab model chat` doesn't automatically connect to the running server when launched with the default values.